### DISCUSSION TITLE
Assets migration from deprecated methods

### DISCUSSION THREAD

**Discussion by satyagraha** [playframework/playframework#11938] [Q&A]
The documentation continues to give examples like:
`GET  /assets/*file                                controllers.Assets.versioned(path="/public", file: Asset)`

However in the code we see:
```
    @deprecated("Inject Assets and use Assets#versioned", "2.6.0")
    override def versioned(file: String) = super.versioned(file)
```
Is there a fully worked example (anywhere) using the recommended runtime injection to serve static (or webjar) files?

**Comment by mkurz** [comment]
https://www.playframework.com/documentation/2.8.x/Migration26#Assets-configuration

All you need to do in the routes files is to remove the `path="/public"` argument.
